style.css

```diff
@@ -1,36 +1,996 @@
-@media screen and (min-width: 375px) {
-   
-    #ham {
-    
-     font-size: 35px;
-     margin-left: 20px;
-    }
-    .main-section {
-        position: relative;
-        background-image: url(Img/background.jpg);
-        background-repeat: no-repeat;
-        background-size: 100%;
-        margin-left: -10px;
-        margin-right: -20px;
-
-    }
-    .middle {
-        color: orangered;
-        width: 50%;
-    }
-    .top {
-        color: orangered;
-    }
-    .main-text {
-        display: flex;
-        flex-direction: column;
-        justify-content: center;
-        align-items: center;
-    }
-    .lower {
-        width: 50%;
-        background-color: rgb(218, 218, 218);
-        font-size: 20px;
-    }
-
-}
\ No newline at end of file
+@import url("https://fonts.googleapis.com/css2?family=Lato&display=swap");
+
+* {
+  padding: 0;
+  margin: 0;
+  text-decoration: none;
+  list-style: none;
+  box-sizing: border-box;
+}
+
+body {
+  overflow-x: hidden;
+  font-family: "Lato", sans-serif;
+}
+
+.d-flex {
+  display: flex;
+}
+
+.fas {
+  color: #272a31;
+}
+
+.removed {
+  display: none;
+}
+
+.d-grid {
+  display: grid;
+  row-gap: 3%;
+}
+
+.middle {
+  background-image: url("Img/orange-color-low-poly-background-triangular-vector-21681635.jpg");
+  background-clip: inherit;
+  -webkit-background-clip: text;
+  color: transparent;
+  background-color: #ec5242;
+}
+
+/* mobile version */
+@media screen and (max-width: 768px) {
+  .mobile-bg {
+    background: url('img/mobile-home-bg.png');
+    background-repeat: no-repeat;
+    background-size: 100%;
+    background-size: cover;
+  }
+
+  .featured-speakers2 {
+    display: none;
+  }
+
+  #dd {
+    display: none;
+  }
+
+  .mobiles-menu {
+    list-style-type: none;
+    overflow: hidden;
+    background: url(img/mobile-home-bg.png);
+    background-repeat: no-repeat;
+    height: 100vh;
+    width: 100%;
+    padding-left: 30px;
+    padding-top: 35px;
+    background-color: white;
+  }
+
+  .mobiles-menu li {
+    padding: 20px;
+  }
+
+  .mobiles-menu li a {
+    font-size: 32px;
+    font-family: "Lato", sans-serif;
+    line-height: 44px;
+    color: #272a31;
+  }
+
+  #horizontal {
+    margin-right: 50px;
+    border-color: #ec5242;
+    border-width: 1px;
+  }
+
+  .main-nav {
+    display: none;
+  }
+
+  #head-h3 {
+    display: none;
+  }
+
+  .navigation {
+    display: none;
+  }
+
+  #ham {
+    font-size: 35px;
+    margin-left: 3%;
+  }
+
+  #hide {
+    margin-left: 3%;
+  }
+
+  .main-section {
+    height: 90vh;
+  }
+
+  .middle {
+    font-family: "Lato", sans-serif;
+    font-size: 45px;
+    font-weight: bolder;
+    color: #ec5242;
+  }
```

Comment:
- [ ] @Zel-hub7 as previously suggested, kindly try and match your headline text to the one on the design given, check the comparison below
![image](https://github.com/Zel-hub7/Capstone-project1/assets/61408860/62b81277-8ae9-4902-aaa8-51271459707c)

Reply:
I have used the same font color and family ..... this text has its own background image mine has also its own which is technically acceptable ... I dont know why you exactly wanted me to clone it ... I think am told to personalize it with any background picture I want.

Reply:
I have tried this time I hope it works i changed the font family to cogooose and added some background picture to the text